app/experimenter/experiments/constants/nimbus.py

```diff
@@ -1,6 +1,60 @@
+from dataclasses import dataclass
+
 from django.db import models
 
 
+@dataclass
+class NimbusTargetingConfig:
+    name: str
+    slug: str
+    description: str
+    targeting: str
+    desktop_telemetry: str
+
+
+TARGETING_ALL_ENGLISH = NimbusTargetingConfig(
```

Comment:
If these configurations originate elsewhere, like in the Nimbus Shared repo, it would be helpful if they included links/references back to the source material and perhaps a 'last update' or something so we know when they were last synced up. There's a pretty abundant use of jargon in the Python code so far, and practically no doc strings (I'm not sure I've seen any at all actually) explaining things at all.

Reply:
Ah no @bbangert this file will be the only source and definition of targeting configs for Nimbus experiments.  This PR is meant to migrate those definitions away from shared.

Reply:
And then they will be removed from the shared repo entirely.

Reply:
Ah, ok. For cross reference, where are the docs on what any of this means? I'm having a tough time reviewing this in as much as 'it looks like valid Python' as I have no reference for this.

Reply:
I think the best documentation for that is here:

https://docs.google.com/document/d/16ekd-cXkJF8_UTgkZ3ufzJFusYz9htd5EE2NpIFFJp4/edit?skip_itp2_check=true&pli=1